completion/services.py

```diff
@@ -76,6 +76,44 @@ def fill_in_run(block_key):
                 completions[candidate] = 0.0
         return completions
 
+    def get_selected_library_children(self, library_node):
```

Comment:
I'm not wild about having to have explicit knowledge of how the library selects children. Does is there any common interface of library and regular blocks that just gets active children? I know if we were working on top of block structures, we could rely on the non-active children having already been filtered out

Reply:
Yeah, I definitely do not care for it either. The only spot I found that is able to handle filtering out the unselected children was the library content block transformer, but we unfortunately can't use that here. This logic is largely based off what happens in that transformer (with some tweaks to work with the object type we have here). And I mentioned this same issue to Nimisha, but attempting to switch  to a block structure format was estimated at 2 devs for 4 weeks. So this is where I landed.